server/sonar-web/src/main/js/apps/quality-profiles/details/__tests__/__snapshots__/ProfileRulesSonarWayComparison-test.js.snap

```diff
@@ -1,29 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
mmmh but I think this one was not... I can't find any `ProfileRulesSonarWayComparison-test.tsx.snap`